resources/assets/scripts/app.js

```diff
@@ -11,6 +11,14 @@ import Router from './util/Router';
 import common from './routes/common';
 import aboutUs from './routes/about';
 
+/**
+ * Helper function for document readiness
+ */
+function ready(fn) {
+  if (document.readyState !== 'loading') return fn();
```

Comment:
I think we should use `window.setTimeout(fn, 0)` so that it runs asynchronously, and I think we should extract this out into a utility function or just [pull it in from npm](https://github.com/nuclei/readyjs).

Reply:
> extract this out into a utility function

Yes 👍 

> pull it in from npm.

No 👎 also that package doesn't use `setTimeout` itself?